soccer/SystemState.cpp

```diff
@@ -86,6 +86,18 @@ void SystemState::drawPolygon(const std::vector<Geometry2d::Point>& pts, const Q
 	dbg->set_color(color(qc));
 }
 
+void SystemState::drawPolygon(const Geometry2d::Polygon& points, const QColor &qc, const QString &layer)
+{
+    const std::vector<Geometry2d::Point>& pts = points.vertices;
```

Comment:
Can we just call the other version of drawPolygon here rather than re-implementing it?